docs/examples/pipeline/query_pipeline_voyageai.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<p>is this the right example for reranking?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I removed the notebook file, i used the same for testing purposes.